docs/source/release_notes/release-process.rst

```diff
@@ -24,12 +24,8 @@ Cutting for Release
 2. Tag the commit: ``git tag -a <version>``.
 3. Push the tag to Github:
    ``git push origin <version>:refs/tags/<version>``.
-4. After a short wait you can publish the artifacts on Bintray by going
+4. After a short wait (when the build at circleCI is successful) the artifacts will be published on BinTray
    to the `AtlasDB
    page <https://bintray.com/palantir/releases/atlasdb/view>`__.
-
-   .. note::
-      You must have admin privledges on Bintray for this to work.
```

Comment:
we should try to protect releases/tags the same way we protect the develop branch (i.e. only the atlas team can write to it)